xknx/xknx.py

```diff
@@ -113,6 +113,8 @@ async def __aexit__(
 
     async def start(self) -> None:
         """Start XKNX module. Connect to KNX/IP devices and start state updater."""
+        if self.connection_config.threaded:
+            await self.connection_manager.register_loop()
```

Comment:
We could move the `threading` argument to `XKNX` instead of having it in the `ConnectionConfig` 🤔

Reply:
I'd leave it there